psydac/ddm/cart.py

```diff
@@ -903,30 +939,47 @@ class InterfaceCartDecomposition(CartDecomposition):
     requests: list of MPI.Request
         the requests of the communications between the cartesian topologies that constucts the interface.
```

Comment:
Same comment as above

Reply:
fixed in [797bf9e](https://github.com/pyccel/psydac/pull/232/commits/797bf9eb03c4f2f616a22fa7b86083b68c8809e9)

Reply:
You fixed the description of `exts`, which is good. But here I was commenting about `requests`

Reply:
ah sorry, it's fixed in [1abfedc](https://github.com/pyccel/psydac/pull/232/commits/1abfedc096b3264444ead795542df2890497b1b4)